packages/ketcher-react/src/script/editor/tool/template.ts

```diff
@@ -590,16 +774,16 @@ function getIgnoredGroupItem(struct: Struct, pasteItems) {
 function getDragCtxItem(
   editor: Editor,
   event,
-  mode: string,
-  mergeItems: MergeItems,
-  findItems
-): { map: string; id: number } | null {
-  if (mode === 'fg') return editor.findItem(event, findItems)
+  mode: MODE,
+  mergeItems: MergeItems | null,
+  findItems: string[]
+): DragContextItem | undefined {
+  if (mode === MODE.ABBREVIATION) return editor.findItem(event, findItems)
   if (mergeItems?.atoms.size === 1 && mergeItems.bonds.size === 0) {
     // get ID of single dst (target) atom we are hovering over
     return { map: 'atoms', id: mergeItems.atoms.values().next().value }
   }
-  return null
+  return undefined
```

Comment:
@yuleicul can you please explain this change

Reply:
The return value of `getDragCtxItem` is assigned to `this.dragCtx.item` whose type is [`DragContextItem | undefined`](https://github.com/epam/ketcher/pull/2391/files#diff-6362aa80fc91114c45092d2c8156d29876eddefa290c3319e18151d352d3e220R111).

Before this PR, the type of `this.dragCtx.item` was like `DragContextItem | undefined | null`. I wanted to simplify  `undefined | null` as `undefined` or `null`. Because we have code like [`delete this.dragCtx.item`](https://github.com/epam/ketcher/pull/2391/files#diff-6362aa80fc91114c45092d2c8156d29876eddefa290c3319e18151d352d3e220R302),  `undefined` is a better solution.

Reply:
Btw, I find this function is deleted by #2408. That's what I wanted to do as well 🤣 I guess there will be some conflicts with #2408, but no worries I can resolve them before merging.